src/main/java/Duke.java

```diff
@@ -1,10 +1,248 @@
+import Duke.exceptions.exceedNum;
+import Duke.tasks.Deadline;
+import Duke.tasks.Event;
+import Duke.tasks.Task;
+import Duke.tasks.Todo;
+
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
+    public static void goodbye() {
+        System.out.println("\nBye.Hope to see you again soon!");
+    }
+
+    public static void greeting() {
+    /*String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    System.out.println("Hello from\n" + logo);*/
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+    }
+
+    public static void checkSum(int sum) throws exceedNum {
+        if (sum==100){
+            throw new exceedNum();
+        }
+    }
+
+    public static void numList(int sum) {
+        if (sum == 1) System.out.println("\tNow you have " + sum + " task in the list.\n");
```

Comment:
The `if-else` block here should follow the structure (see screenshot below):

![Screenshot 2020-09-15 at 1 14 31 AM](https://user-images.githubusercontent.com/59989652/93117102-e30bb280-f6f0-11ea-8ccb-6717abb6c5ee.png)

Reply:
Thank you! I've changed to the correct style.